modules/ROOT/pages/telemetry-exporter.adoc

```diff
@@ -0,0 +1,144 @@
+= Export Anypoint Platform Data to Third-Party Apps
+
+The Telemetry Exporter feature enables you to export xref:access-management::audit-logging.adoc[_audit logs_] to third-party observability and analytics apps, such as Azure Monitor, Datadog, and Splunk.
+//trace data should be a link once the topic exists
+
+To use the Telemetry Exporter feature, you must create a _connection_ with a third-party app to authorize Anypoint Platform to share data. You must then create a _configuration_ to specify the frequency of data transfers. You can also edit and delete connections and configurations.
```

Comment:
Configuration does not specify the frequency of data transfers. It specifies routing logic on which business group to export to which connections.

Reply:
```suggestion
To use the Telemetry Exporter feature, you must create a _connection_ with a third-party app to authorize Anypoint Platform to share data. You must then create a _configuration_ to specify which business groups allow content to be exported to connections. You can also edit and delete connections and configurations.
```